src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/DiaryOverviewTableViewController.swift

```diff
@@ -16,14 +17,20 @@ class DiaryOverviewTableViewController: UITableViewController {
 		onEditContactPersonsButtonTap: @escaping () -> Void,
 		onEditLocationsButtonTap: @escaping () -> Void
 	) {
-		self.viewModel = DiaryOverviewViewModel(diaryService: diaryService)
+		self.diaryService = diaryService
```

Comment:
Maybe we should rename our services to ViewModel?

Reply:
Let's discuss this once we've used it everywhere, might actually make sense to call this a view model